src/App.js

```diff
@@ -1,25 +1,46 @@
-import logo from './logo.svg';
-import './App.css';
+import { Fragment, useState, useEffect } from "react";
+import { getUser, getFolder } from "./api/api";
+import "./global.css";
+import Layout from "./components/Layout/Layout";
+import FolderInfo from "./components/FolderInfo/FolderInfo";
+import SharePage from "./pages/SharePage/SharePage";
+import SearchBar from "./components/SearchBar/SearchBar";
+import CardList from "./components/CardList/CardList";
+import OnlyCard from "./components/OnlyCard/OnlyCard";
```

Comment:
P3:
component 폴더에서 각 항목별로 개별 import 를 하니 가독성이 좀 떨어지는 것 같아요.
아래와 같은 식으로 export 하고 import 할 수 있습니다.

```suggestion
import { CardList, FolderInfo, SearchBar, CardList, OnlyCard } from "./components"; // component들은 한 폴더에서 여래개 export 될 수 있으니 named export 활용
import SharePage from "./pages/SharePage/SharePage"; // page는 한개만 export 될 가능성이 크니 export default 활용
```